app/models/letter.rb

```diff
@@ -52,4 +52,3 @@ def company_name
     values.third_party_company_name if values.third_party_company_name.present?
   end
 end
-
```

Comment:
There should be a carriage return at the end of every file, I don't think this needs to be removed?

Reply:
Strange, there is a carriage return at the end of my copy of the file.